DatadogInternal/Sources/Telemetry/Telemetry.swift

```diff
@@ -58,11 +58,32 @@ public struct ConfigurationTelemetry: Equatable {
     public let useWorkerUrl: Bool?
 }
 
+public struct MetricTelemetry {
+    /// The default sample rate for metric events, applied in addition to the telemetry sample rate (20% by default).
```

Comment:
The default telemetry sample rate is 20% and the default metric sample rate is 15%, correct? The comment might be a bit confusing here.

Reply:
Yes, it's exactly like you said. And this is very fair call, I updated the comment as suggested 👍.